lgc/patch/FragColorExport.cpp

```diff
@@ -99,224 +99,170 @@ namespace lgc {
 FragColorExport::FragColorExport(LLVMContext *context) : m_context(context) {
 }
 
+// =====================================================================================================================
+// Split a possible vector value, if there are less than 4 elements, append with null value.
```

Comment:
maybe something like: `Extracts all vector elements. Returns an array of vector elements, padded with null values at the end when the input vector has less than 4 elements.`

Reply:
I would like to rename the function as `extractElements()`, the comment will be something like `Extract all the scalar elements of a scalar or vector type (with at-most four elements) input. Return the array of elements, padded with null values at the end when the input vector has less than 4 elements`. I am not sure does this sounds OK to you and others?

Reply:
sounds good to me